.github/workflows/rebuild-clade-i.yaml

```diff
@@ -0,0 +1,60 @@
+name: Rebuild clade-I
+
+on:
+  repository_dispatch:
+    types:
+      - rebuild
+      - rebuild_clade-i
+
+  workflow_dispatch:
+    inputs:
+      trial_name:
+        description: "If set, result will be at nextstrain.org/staging/trial/${trial_name}/${auspice_name}"
+        required: false
+
+jobs:
+  set_config_overrides:
+    runs-on: ubuntu-latest
+    steps:
+      - id: config
+        name: Set config overrides
+        env:
+          TRIAL_NAME: ${{ inputs.trial_name }}
+        run: |
+          config=""
+          if [[ "$TRIAL_NAME" ]]; then
+            config+="--config"
+            config+=" deploy_url='s3://nextstrain-staging/'"
+            config+=" auspice_prefix='"$TRIAL_NAME"'"
+          fi
+
+          echo "config=$config" >> "$GITHUB_OUTPUT"
+    outputs:
+      config_overrides: ${{ steps.config.outputs.config }}
+
+  rebuild_clade_i:
+    needs: [set_config_overrides]
+    permissions:
+      id-token: write
+    uses: nextstrain/.github/.github/workflows/pathogen-repo-build.yaml@master
+    secrets: inherit
+    with:
+      runtime: conda
+      env: |
+        CONFIG_OVERRIDES: ${{ needs.set_config_overrides.outputs.config_overrides }}
+        GITHUB_RUN_ID: ${{ github.run_id }}
+        SLACK_CHANNELS: ${{ inputs.trial_name && vars.TEST_SLACK_CHANNEL || vars.SLACK_CHANNELS }}
+        BUILD_DIR: phylogenetic
+        BUILD_NAME: clade-i
+      run: |
+        nextstrain build \
+          --detach \
+          --no-download \
```

Comment:
The `--detach` and `--no-download` flags are only relevant for the `aws-batch` runtime and will raise an error if not used with the `aws-batch` runtime.

Reply:
Thanks! [Fixed up.](https://github.com/nextstrain/mpox/compare/3b19cc9be008070140c0a617c030487bf9e9e9ac..243f6c5d7ccd80c1631161cfef12ff353cbe9a24)